packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -8,10 +8,12 @@ import { MemberAlready } from '../onboarding/MemberAlready';
 import { useAuthContext } from '../../contexts/AuthContext';
 import { BottomBannerContainer } from '../banners';
 import { ButtonVariant } from '../buttons/common';
+import { Image } from '../image/Image';
+import { cloudinaryAuthBannerBackground as bg } from '../../lib/image';
 
 const Section = classed('div', 'flex flex-col');
-export const authGradientBg =
-  'bg-background-default bg-gradient-to-l from-theme-overlay-active-cabbage from-0% to-theme-overlay-active-onion to-100%';
+const laptopBg = bg.replace('/f_auto', '/c_auto,g_center,w_1440/f_auto/');
+const desktopBg = bg.replace('/f_auto', '/c_auto,g_center,w_1920/f_auto/');
```

Comment:
The duplication is also not needed, but again wouldn't even do this transform I don't think.